packages/inter-protocol/src/proposals/upgrade-vaults.js

```diff
@@ -149,7 +139,7 @@ export const upgradeVaults = async (
     });
 
     const upgradeResult = await E(kit.adminFacet).upgradeContract(
-      bundleID,
+      VaultFactoryBundle.bundleID,
```

Comment:
that name is kinda misleading.

I went "whoa! we're putting the whole text of the vault factory contract in there??!?" but then I realized no, it's just a small data structure with a `.bundleID`. The relevant type is `ManifestBundleRef`. A name based on that might be nice: `vaultsBundleRef`.